README.md

```diff
@@ -1,5 +1,48 @@
-# curriculum-cr-training-contact-form
+# Portfolio contact form
 
-This repo host an initial code for the tasks performed during the Code Reviewers training.
+> This repo host an initial code for the tasks performed during the Code Reviewers training.
+
+## Built With
+
+- HTML
+- CSS
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+### Install
+- Clone this git repository.
+   ```sh
+       git clone https://github.com/sinansevgi/contact-form.git
+   ```
+- Open the terminal in the project directory using 'cd contact-form'.
+- Navigate `index.html`with your browser.
+
+## Authors
+
+👤 **Sinan Sevgi**
+
+- GitHub: [@sinansevgi](https://github.com/sinansevgi)
+- LinkedIn: [@sinansevgi](https://www.linkedin.com/in/sinan-s-52559437/)
+- Twitter: [@nativeofcybers1](https://twitter.com/nativeofcybers1)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](issues/).
```

Comment:
- [ ] **The link for issues page is broken. Please fix this.**

Broken links in our README reflect imperfection and make our repo unprofessional. This might affect your impression when someone views your repository. Please fix the broken link.